test/net/integration/slaac/test.py

```diff
@@ -26,7 +26,7 @@ def Slaac_test(trigger_line):
   print(color.INFO("<Test.py>"), "Trying to ping")
   time.sleep(1)
   try:
-    command = ["ping6", "-I", "bridge43", ip_string.rstrip(), "-c",
+    command = ["ping", "-I", "bridge43", ip_string.rstrip(), "-c",
```

Comment:
```suggestion
    command = ["ping", "-6", "-I", "bridge43", ip_string.rstrip(), "-c",
```

To still use ipv6 you could add `-6` as a parameter to ping.

Reply:
Yea, turns out it works for ipv6 without it - so I think this is only necessary to exclude IPv4.

Reply:
Yes, the only benefit would be that it would fail if it accidentally was pinging an IPv4 address (which may not be likely here)

```
$ ping -6 127.0.0.1
ping: 127.0.0.1: Address family for hostname not supported
```